docs/installation/linux/debian.md

```diff
@@ -36,6 +36,26 @@ Docker is supported on the following versions of Debian:
 
      $ uname -r
 
+ Additionally, for users of Debian Wheezy, backports must be available. To enable backports in Wheezy:
+ 
+ 1. Log into your machine and open a terminal with `sudo` or `root` privileges.
+ 
+ 2. Open the `/etc/apt/sources.list.d/backports.list` file in your favorite editor.
+
+     If the file doesn't exist, create it.
+ 
+ 3. Remove any existing entries.
+ 
+ 4. Add an entry for backports on Debian Wheezy.
+
+     An example entry:
+
+    deb http://http.debian.net/debian wheezy-backports main
```

Comment:
Can you indent this line with 4 more spaces? Currently it doesn't render as a "code" block, indenting it with 4 extra spaces will;

before:

<img width="691" alt="screen shot 2016-05-25 at 23 49 46" src="https://cloud.githubusercontent.com/assets/1804568/15557508/b84eb2f4-22d3-11e6-8e48-56db746d1b4d.png">

after:

<img width="719" alt="screen shot 2016-05-25 at 23 51 00" src="https://cloud.githubusercontent.com/assets/1804568/15557510/c38ec51e-22d3-11e6-98fb-e06b0e7ca81d.png">